src/debuginfo.cpp

```diff
@@ -1039,7 +1040,7 @@ extern "C" void jl_register_fptrs(uint64_t sysimage_base, void **fptrs, jl_metho
 template<typename T>
 static inline void ignoreError(T &err)
 {
-#if JL_LLVM_VERSION >= 30900 && !defined(NDEBUG)
+#if JL_LLVM_VERSION >= 30900 && !defined(JL_NDEBUG)
```

Comment:
I was hoping replacing all with `JL_NDEBUG` in `*.cpp` can be less confusing to readers....... It should be roughly the same since I unset `JL_NDEBUG` if `NDEBUG` isn't set. Which one should I do?